run_kraken.py

```diff
@@ -387,11 +387,15 @@ def main(cfg):
             logging.info(f"telemetry data will be stored on s3 bucket folder: {telemetry_request_id}")
             logging.info(f"telemetry upload log: {safe_logger.log_file_name}")
             try:
-                telemetry.send_telemetry(config["telemetry"], telemetry_request_id, chaos_telemetry)
-                safe_logger.info("archives download started:")
-                prometheus_archive_files = telemetry.get_ocp_prometheus_data(config["telemetry"], telemetry_request_id)
-                safe_logger.info("archives upload started:")
-                telemetry.put_ocp_prometheus_data(config["telemetry"], prometheus_archive_files, telemetry_request_id)
+                telemetry_json = telemetry.send_telemetry(config["telemetry"], telemetry_request_id, chaos_telemetry)
+                if telemetry_json:
+                    telemetry_obj = ChaosRunTelemetry(json.loads(telemetry_json))
+                    logging.info(f"Telemetry data:\n{telemetry_obj.to_json()}")
```

Comment:
It might be useful if we capture and print the json even when telemetry is disabled i.e 
if telemetry is enabled we ship it to the centralized service location and print it as part of stdout
if telemetry is disabled we just print the json as part of stdout

Thoughts?

Reply:
:+1: